### DISCUSSION TITLE
Welcome to Winget-AutoUpdate-Intune Discussions!

### DISCUSSION THREAD

**Discussion by Weatherlights** [Weatherlights/Winget-AutoUpdate-Intune#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chiujiewei** [comment]
I have been using this, it works really well, really grateful to the developer for this, I'm just wondering if this will be updated to a newer version? I see Romanitho's version is now on v3.0.0-0.

Thanks!
